.env.example

```
@@ -5,6 +5,8 @@ NUXT_V4_PATH=
 NUXT_EXAMPLES_PATH=
 # Used when pre-rendering the docs for dynamic OG images
 NUXT_PUBLIC_SITE_URL=
+# Enables runtime OG image generation (generate with: npx nuxt-og-image generate-secret)
+NUXT_OG_IMAGE_SECRET=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NUXT_OG_IMAGE_SECRET key should go before the NUXT_PUBLIC_SITE_URL key

(UnorderedKey)